src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -907,6 +907,9 @@
         - name: --grafana-resource-id
           type: string
           short-summary: Resource ID of the Azure Managed Grafana Workspace
+        - name: --grafana-resource-id
+          type: bool
+          short-summary: Enable Windows Recording Rules when enabling the Azure Monitor Metrics addon
```

Comment:
wrong name? should be --enable-windows-recording-rules, right?

Reply:
yes! updated.